lib/iris/tests/unit/constraints/test_Constraint_equality.py

```diff
@@ -67,38 +63,38 @@ def test_funcs_differ(self):
         self.assertNotEqual(c1, c2)
 
     def test_coord_names_same(self):
-        c1 = Constraint(coord=3)
-        c2 = Constraint(coord=3)
+        c1 = Constraint(some_coordname=3)
+        c2 = Constraint(some_coordname=3)
         self.assertEqual(c1, c2)
 
     def test_coord_names_differ(self):
-        c1 = Constraint(coord=3)
-        c2 = Constraint(coord2=3)
+        c1 = Constraint(coordname_A=3)
+        c2 = Constraint(coordname_B=3)
         self.assertNotEqual(c1, c2)
 
     def test_coord_values_differ(self):
-        c1 = Constraint(coord=3)
-        c2 = Constraint(coord=4)
+        c1 = Constraint(coordname=3)
+        c2 = Constraint(coordname=4)
```

Comment:
This is a bit of a nitpick, but I think this would be better as `some_coordname` like above.